src/chrome/content/rules/GayAgenda.lgbt.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="GayAgenda.lgbt">
+	<target host="gayagenda.lgbt" />
+	<target host="www.gayagenda.lgbt" />
```

Comment:
See also:
```
3gang.ch
cominginn.ch
gaybern.ch
ha-bern.ch

Reply:
@J0WI Quite a lot of domains since each of them may have subdomains.

Reply:
@J0WI Done.